python/cudf/cudf/_lib/cpp/io/types.pxd

```diff
@@ -36,7 +36,7 @@ cdef extern from "cudf/io/types.hpp" \
         ARROW_RANDOM_ACCESS_FILE \
             "cudf::io::io_type::ARROW_RANDOM_ACCESS_FILE"
         VOID "cudf::io::io_type::VOID"
-        USER_SINK "cudf::io::io_type::USER_SINK"
+        USER_IMPLEMENTED "cudf::experimental::io::io_type::USER_IMPLEMENTED"
```

Comment:
Is USER_IMPLEMENTED in the cudf::experimental namespace? Looks like it should still be cudf::io::io_type  to me
```suggestion
        USER_IMPLEMENTED "cudf::io::io_type::USER_IMPLEMENTED"
```

Reply:
Good catch. Looks like a botched a merge here.